.travis.yml

```diff
@@ -32,12 +27,51 @@ matrix:
         - docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 fiware/orion -dbhost mongodb
       before_script:
         - npm run lint -C ./validator
+       
+      script:
+         - node ./validator/validate.js -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p specs -c true
+
+    - stage: test
+      name: "Unit Tests 3.6"
+      python: 3.6
```

Comment:
could it be 3.7?

Reply:
I use 3.7.2, but not all harvesters  use it

Reply:
Fixed 3ad53be + 20f48dd - Travis offers 3.7.1 (using an official workaround)